data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,63 @@ mission "Arachi Cartoonist 2"
+			`	"Don't know why they bother with that, it's not like I hit you too hard," the Alpha says. "I guess I owe you an apology, though. I've given up on fighting humanity left and right, really, I just got a bit jumpy when I spotted you."`
+			choice
+				`	"Who are you, exactly? How did you even get here?"`
+				`	"What are you doing here in the Coalition?"`
+			`	"Where do I begin? Well, for starters, name's Alitis. Was Athis before, but I took a Saryd name after I got here. Back in the 2800s the crew I was on found some aliens somewhere far north of Earth, but we didn't get to get much from them as they were fighting another group of aliens, ones who had jump drives. We didn't want any part of the fighting, as the battle wouldn't be favorable to us, but we managed to loot a jump drive from one of the aliens. After that we started exploring disconnected systems to the east. Eventually, we joined an enclave in the Dirt Belt, but it didn't take long for the Navy to find us there. While evacuating, some of the others wanted to head back east, but I knew better, so I banked on heading further west, and eventually got here."`
+			choice
+				`	"You 'knew better?' What do you mean?"`
+				`	"What, did you have a change of heart once you got here?"`
+			`	He sits on the ground for a while, thinking. "I was one of those that fought humanity after the first Turf War. I was ambushed, cornered many times, fear wasn't a stranger to me. After we stole that jump drive and ventured east, however, someone or... something rather, some voice spoke to me. It was the first time I felt scared. After that, I wasn't inclined to go back east with the others, no matter how silly a reason they thought it was. When I got here, me and my crew helped the Heliarchs with translations for humans, and I told them about humanity and the Alphas. They took me in, but demanded my jump drive, so I handed it over, damn thing got me in more trouble than it was worth anyway."`
+			choice
+				`	"So where are the other Alphas that came with you?"`
+					goto others
+				`	"What did the voice say?"`
+			`	He forces a chuckle out. "You got here, so you have a jump drive, I assume? Go on, head east and maybe it'll speak to you too. Find out if it classifies you as 'disposable' as well."`
+				goto end
+			label others
+			`	"Others? Oh, you misunderstood me. I took a crew of brigands and pirates, they're all long dead now, most of old age. I'd say they made the right choice in coming with me, no matter how much they protested that the aliens here were too alien."`
+			label end
+			`	The Heliarchs inform you the medic has arrived, and have them treat your bruises. Although the Alpha was right in that you weren't injured much, the Heliarchs insist on cleaning up your bruises. Once they're done, an agent escorts you back to your ship.`
```

Comment:
You must've bruised quick for this to happen.

Reply:
?

Reply:
https://www.thoughtco.com/what-is-a-bruise-4160373 <-- This website says that a red bruise can form right after an injury, so I could see this happening.